src/main/kotlin/stringPlusCalculator/CustomStringParser.kt

```diff
@@ -0,0 +1,23 @@
+package stringPlusCalculator
+
+class CustomStringParser {
+    companion object {
+        fun parse(expressionInput: String): List<String> {
+            val stringExpression = findStringExpression(expressionInput)
+            val customStringParser = findCustomStringParser(expressionInput)
+
+            return when {
+                customStringParser.isNullOrBlank() -> listOf(stringExpression)
```

Comment:
nullable하지 않기 때문에 다음으로 대체할 수 있습니다.
```suggestion
                customStringParser.isBlank() -> listOf(stringExpression)
```